### PR TITLE
17

### PR DESCRIPTION
**Pull request by felixwiberg**
$python make_sheet.py path-to-model-folder
the above command compiles a pdf document with network layer data and a loss-over-time figure inside path-to-model-folder
depencencies : latex with inputenc
if latex is not found; a .tex-file is generated instead to be compiled online.

### CODE REVIEW

File: rirnet/make_sheet.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import torch as th
+import numpy as np
+import pprint
+import os
+import subprocess
+import sys
+import collections
+import json
```
Comment (felixwiberg) [inline rirnet/make_sheet.py]: nope

File: rirnet/make_sheet.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+def main(model_dir):
+    sys.path.append(model_dir)
+    net = import_module('net')
+    model = net.Net()
+    args = model.args()
+    input_size = get_training_data_shape(model, args)
+
+    use_cuda = not args.no_cuda and th.cuda.is_available()
+    device = th.device("cuda" if use_cuda else "cpu")
```
Comment (felixwiberg) [inline rirnet/make_sheet.py]: use same convention throughout project
